docs/get-started/running-nodes.mdx

```diff
@@ -4,306 +4,218 @@ title: 'Running Etherlink Nodes'
+        "mode": "observer" }
+      ```
+
+1. Activate the `unsafe-pvm-patches` setting, which is required for Etherlink, and add the location of the preimages by adding these settings to the `config.json` file, as in this example:
+
+   ```json
+   { "smart-rollup-address": "sr1Ghq66tYK9y3r8CC1Tf8i8m5nxh8nTvZEf",
+     "smart-rollup-node-operator": {}, "fee-parameters": {},
+     "unsafe-pvm-patches": [
+       { "increase_max_nb_tick": "50_000_000_000_000" }
+     ],
+     "pre-images-endpoint": "https://snapshots.eu.tzinit.org/etherlink-mainnet/wasm_2_0_0",
+     "mode": "observer" }
+   ```
+
+   Similar to the Rust keyword, `unsafe` here means to be used with a good understanding of the feature.
+   Etherlink uses this setting with care and using it is safe in this context.
+
+1. Start the Smart Rollup node in observer mode by running this command and using the RPC endpoint of a layer 1 node that is running in archive mode:
+
+   ```bash
+   octez-smart-rollup-node --endpoint https://rpc.tzkt.io/mainnet run \
+     --data-dir $sr_observer_data_dir \
+     --apply-unsafe-patches
+   ```
+
+   As in this example, you can use a public layer 1 RPC node for testing purposes.
+   For production use, you must use a layer 1 node that is running in archive mode and provides access to its RPC endpoints.
+
+   The `--apply-unsafe-patches` setting is required only the first time that you start the node.
+   After that you can omit it.
+   After the first run, you can also remove the `unsafe-pvm-patches` field from the configuration file.
+
+   After the Smart Rollup node has caught up with the current state via the archive node, you can safely connect it to a rolling layer 1 node.
+
+1. Verify that the Smart Rollup node is running by querying it.
+For example, this query gets the latest block:
+
+   ```bash
+   curl -s http://localhost:8932/global/block/head
+   ```
+
+### Running an Etherlink node
+
+To run an Etherlink node in observer mode from the Etherlink genesis block, you need:
+
+- Access to a Smart Rollup node tracking the state of Etherlink, which you created in the previous section
+- The Etherlink installer kernel (`installer.hex` file), which you can build yourself as described in [Building the Etherlink kernel](#building-the-etherlink-kernel) or download here: [installer.hex](/files/installer.hex)
+
+Follow these steps to run the Etherlink node:
+
+1. Set the `sr_node_observer_rpc` environment variable to the URL to the Smart Rollup node you set up in the previous section.
+1. Set the `evm_observer_dir` environment variable to the directory where the node should store its local data.
+1. Initialize the node by running this command:
+
+   ```bash
+   octez-evm-node init config --devmode \
+     --data-dir $evm_observer_dir --rollup-node-endpoint $sr_node_observer_rpc \
+     --preimages-endpoint https://snapshots.eu.tzinit.org/etherlink-mainnet/wasm_2_0_0 \
+     --evm-node-endpoint https://node.mainnet.etherlink.com \
+     --force
```

Comment:
I would remove the `--force` it's only if the config already existed in the data-dir